### PR TITLE
Profile needs tabs

### PR DESCRIPTION
**Pull request by Linkster700**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,80 +1,103 @@
...
-  <div class="col-md-3">
-    <center>
-      <img id="avatar" src="http://www.gravatar.com/avatar/<%= @user.email %>?s=200px" />
-    </center>
-    <br>
-    <br>
-    <div class="panel panel-default">
+<div class="container">
+  <div class="col-md-4">
+      <center>
```
Comment (bammons) [inline app/views/users/show.html.erb]: That looks like 4 spaces and not 2

Comment (Linkster700) [inline app/views/users/show.html.erb]: done

**Comment by bammons** [comment]
Need to fix spacing in html. Future reference branch name should be task number on scrumwise. The commit message needs to reference the issue that is associated with the scrumwise task

:-1:

**Comment by bammons** [comment]
:+1:

**Comment by kdorer** [comment]
+1

**Pull request merged**